src/content/en/2022/capabilities.md

```diff
@@ -1,13 +1,13 @@
 ---
 #See https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Authors'-Guide#metadata-to-add-at-the-top-of-your-chapters
 title: Capabilities
-#TODO - Review and update chapter description
 description: Capabilities chapter of the 2022 Web Almanac covering brand-new, powerful web platform APIs that give web apps access to hardware interfaces, enhance web-based productivity apps, and more.
 authors: [MichaelSolati]
 reviewers: [webmaxru, hemanth, beaufortfrancois, tomayac, christianliebel]
```

Comment:
@MichaelSolati can you confirm these are correct and listed in descending order of contribution?